src/sentry/api/endpoints/rule_snooze.py

```diff
@@ -0,0 +1,96 @@
+import datetime
+
+from rest_framework import serializers, status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import audit_log, features
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectAlertRulePermission, ProjectEndpoint
+from sentry.api.exceptions import ConflictError
+from sentry.api.serializers import serialize
+from sentry.api.serializers.rest_framework.base import CamelSnakeSerializer
+from sentry.incidents.models import AlertRule
+from sentry.models import Rule, RuleSnooze
+
+
+class RuleSnoozeSerializer(CamelSnakeSerializer):
```

Comment:
nit: we're trying to call serializers that validate input `Validators`, e.g. https://github.com/getsentry/sentry/blob/b67df8ce3aa7590658f3eb49afceb762dffb6bf5/src/sentry/monitors/validators.py#L138